scripts/install.in

```diff
@@ -18,7 +18,7 @@ cleanup() {
 trap cleanup EXIT INT QUIT TERM
 
 require_util() {
-    type "$1" > /dev/null 2>&1 || command -v "$1" > /dev/null 2>&1 ||
+    command -v "$1" > /dev/null 2>&1 || command -v "$1" > /dev/null 2>&1 ||
```

Comment:
It was already doing `command -v` as a fallback, so now it's coding `command -v || command -v` which is redundant.

Reply:
Thanks for pointing out the redundancy, it's removed.